common/shell-local/config.go

```diff
@@ -5,6 +5,7 @@ package shell_local
 import (
 	"errors"
 	"fmt"
+
 	// "log"
```

Comment:
Should we just remove that line here instead ?
```suggestion
```

Reply:
Good catch. I should be better about spot checking goimport changes. I'll review and update the groupings and remove any commented out imports.